src/main/kotlin/no/nav/familie/ef/mottak/api/dto/SøknadDto.kt

```diff
@@ -1,3 +1,6 @@
 package no.nav.familie.ef.mottak.api.dto
 
-data class SøknadDto(val text: String)
+data class SøknadDto(
```

Comment:
Håper vi til uka kan se på dataformat for innsending, mellom appene og i database :)